docs/source/index.rst

```diff
@@ -553,6 +553,7 @@ Flax), PyTorch, and/or TensorFlow.
     model_doc/auto
     model_doc/bart
     model_doc/barthez
+    model_doc/bartpho
```

Comment:
Don't forget to add your model to the main README (in alphabetical order) then run `make fix-copies` to update this file as well.

Reply:
I tried adding my other two models BERTweet and PhoBERT to the main README when making first commits 1+ years ago, but got rejections. Reason: the only thing I contributed is the new language/domain-specific tokenizers (not modeling). 
Can I now add BARTpho as well as BERTweet (EMNLP 2020 Demonstrations) and PhoBERT (EMNLP 2020 Findings) into the main README file?